lib/provider/camunda/CamundaPropertiesProvider.js

```diff
@@ -140,6 +143,46 @@ var getListenerLabel = function(param, translate) {
 var PROCESS_KEY_HINT = 'This maps to the process definition key.';
 var TASK_KEY_HINT = 'This maps to the task definition key.';
 
+function getIdOptions(element) {
+  if (is(element, 'bpmn:Process')) {
+    return { description: PROCESS_KEY_HINT };
+  }
+
+  if (is(element, 'bpmn:UserTask')) {
+    return { description: TASK_KEY_HINT };
+  }
+}
+
+function getProcessOptions(element) {
+  if (is(element, 'bpmn:Participant')) {
+    return { processIdDescription: PROCESS_KEY_HINT };
+  }
+}
+
+function createElementTemplateGroups(element, bpmnFactory, canvas, elementTemplates, modeling, replace, selection, translate) {
```

Comment:
I like a lot that we now separate the template groups from the "normal" state. 👏